analysis/src/CreateInterface.ml

```diff
@@ -31,6 +31,58 @@ module SourceFileExtractor = struct
       !res)
 end
 
+module AttributesUtils : sig
+  type t
+
+  val separate : string list -> t
+
+  val contain : string -> t -> bool
+
+  val concat : string -> t -> string
+end = struct
+  type t = string list
+
+  let separate lines =
+    lines
+    |> List.fold_left
+         (fun acc line ->
+           let trimmedLine = String.trim line in
+           if trimmedLine |> String.length > 0 && String.get trimmedLine 0 = '@'
+           then (
+             let attrDecIdxSeparator = ref (-1) in
+             let i = ref 1 in
+             let lineLength = String.length line in
+
+             while !attrDecIdxSeparator = -1 do
+               if !i < lineLength then
+                 let idx = !i in
+                 let currentChar = line.[idx] in
```

Comment:
How do we know that `line[1]` exists?

Reply:
I think it can be made more obvious.

Reply:
Oh, thanks. I think I get it. I'll do it.